src/main/java/net/mcreator/io/net/analytics/GoogleAnalytics.java

```diff
@@ -54,12 +54,9 @@ private String getGATrackURL(String hitType, Map<String, Object> payload) {
 		payload.put("t", hitType);
 
 		for (Map.Entry<String, Object> entry : payload.entrySet()) {
-			try {
-				if (entry.getValue() != null)
-					actionRequestURL.append("&").append(entry.getKey()).append("=")
-							.append(URLEncoder.encode(entry.getValue().toString(), "UTF-8"));
-			} catch (UnsupportedEncodingException ignored) {

```

Comment:
I would keep the catch here, just to be safe. Maybe we should just add some logging here

Reply:
The catch is redundant, since the method doesn't throw `UnsupportedEncodingException` anymore

Reply:
It does: https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/net/URLEncoder.html#encode(java.lang.String,java.lang.String)

Reply:
That's the one where you pass two strings, here we directly pass the charset

Reply:
Correct :)